src/models/koder.js

```diff
@@ -61,6 +61,11 @@ const koderSchema = new Schema({
         : dayjs().add(100, 'year')
     }
   },
+  deactivationReason: {
+    type: String,
+    enum: ['unpaid', 'unresponsive', 'expelled', 'desertion'],
+    required: () => !!this.isActive
```

Comment:
Aqui deberia ser required cuando isActive s false no true
```suggestion
    required: () => !this.isActive
```

Reply:
Lo puse directamente igual a false ya que si lo ponia asi ! me segui diciendo que era requerido